tests/unit/util/CMakeLists.txt

```diff
@@ -16,10 +16,24 @@ blt_add_test(
   NAME exception_tests
   COMMAND exception_tests)
 
+set (allocation_map_tests_depends
+  umpire gtest)
+
+if (ENABLE_CUDA)
+  set (allocation_map_tests_depends
```

Comment:
Was this a regression?  How did this new dependency come about and how is this related to this specific PR?

Reply:
I don't know. I saw a HIP build error on the previous commit and added it. BLT isn't carrying over the dependencies from umpire correctly, because e.g. `cuda` implies `cuda_runtime`. It's complicated...